feature_tests/src/structs.rs

```diff
@@ -1,11 +1,20 @@
 #[diplomat::bridge]
 pub mod ffi {
+    use diplomat_runtime::DiplomatStr16;
+
     use crate::imports::ffi::ImportedStruct;
+    use std::sync::Mutex;
 
     #[diplomat::opaque]
     #[diplomat::transparent_convert]
     pub struct Opaque(String);
 
+    #[diplomat::opaque]
+    pub struct OtherOpaque(Mutex<String>);
```

Comment:
nit: let's give this a more descriptive name for its purpose, maybe MutexedOpaque?

Reply:
I called this OpaqueMutexedString